data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/ApplicableRangeAction.java

```diff
@@ -20,4 +20,6 @@
 
     // The setpoint is computed by an optimiser.
     void apply(Network network, double setpoint);
+
+    NetworkElement getNetworkElement();
```

Comment:
I think you can declare `List<NetworkElement> getNetworkElements();` directly here

Reply:
I made some changes according to your review. Could you check it again?
One side effect is the roundTripTest of JsonImportExportTest stops working. If you have a quick fix it would be welcome; otherwise I am going to check the JsonImportExport..

Reply:
I tried to check quickly on the Json test but I didn't find an explanation...